cluster-scope/base/apiextensions.k8s.io/customresourcedefinitions/kfdefs.kfdef.apps.kubeflow.org/customresourcedefinition.yaml

```diff
@@ -17,10 +17,14 @@ spec:
       description: KfDef is the Schema for the kfdefs API
       properties:
         apiVersion:
-          description: 'APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
+          description: 'APIVersion defines the versioned schema of this representation
```

Comment:
I think this was formatted by accident. Can we please not change the copies of upstream CRD manifests?